packages/next/src/components/HeadlessApp.tsx

```diff
@@ -47,6 +48,13 @@ export type HeadlessAppProps = {
 	 */
 	useYoastHtml?: boolean;
 
+	/**
+	 * If true, will automatically load yoast seo metadata into the head
+	 *
+	 * @default true
+	 */
+	handleYoast?: boolean;
```

Comment:
@nicholasio This flag when true, will mean that `useSeo()` will always be populated?

Reply:
`useSeo` is always populated regardless of this flag. This flag merely loads the `Yoast` component which puts the metatags on the head.

This would be for a situation where the project wants to customize how the SEO metatags are handled.

Reply:
Sweet 👍